viahtml/stats.py

```diff
@@ -0,0 +1,108 @@
+"""uWSGI -> New Relic metrics collection logic."""
+
+from collections import defaultdict
+from logging import getLogger
+
+import requests
+from requests import RequestException
+
+LOG = getLogger(__name__)
+
+
+# pylint: disable=missing-yield-doc,missing-yield-type-doc
```

Comment:
What do you want from me `pylint`?

Reply:
In LMS we've globally disabled PyLint rules about missing docstrings or missing parts of docstrings (returns and types etc). I think we should probably do that throughout our apps